src/sync/sync_session.hpp

```diff
@@ -198,6 +198,9 @@ class SyncSession : public std::enable_shared_from_this<SyncSession> {
     // not make the session reconnect.
     void set_multiplex_identifier(std::string multiplex_identity);
 
+    // Inform sync that a local transaction has been commited.
+    void nonsync_transact_notify(VersionID::version_type);
```

Comment:
Rather than making this public the new call to it should continue to go through Internal::nonsync_transact_notify().